docs/CONFIG-VARS.md

```diff
@@ -161,56 +161,56 @@ When setting V4_CFG_MANAGE_STORAGE to true, A new storage classes will be create
+| V4_CFG_CM_CERTIFICATE_DURATION | Certificate time to expiry in hours | string | 17531h | false | | viya |
+| V4_CFG_CM_CERTIFICATE_ADDITIONAL_SAN_DNS | A list of space separated, additional SAN DNS entries, specific to your ingress architecture, that you want added to certificates issued by the sas-viya-issuer.  For example, the aliases of an external load balancer | string | | false | | viya |
+| V4_CFG_CM_CERTIFICATE_ADDITIONAL_SAN_IP | A list of space separated, additional SAN IP addresses, specific to your ingress architecture, that you want added to certificates issued by the sas-viya-issuer.  For example, the IP address of an external load balancer | string | | false | | viya |
 
 ## Postgres
 
 | Name | Description | Type | Default | Required | Notes | Actions |
 | :--- | ---: | ---: | ---: | ---: | ---: | ---: |
-| V4_CFG_POSTGRES_TYPE | Postgres installation type | string | | true | [internal,external] | vdm |
+| V4_CFG_POSTGRES_TYPE | Postgres installation type | string | | true | [internal,external] | viya |
 
 ### External Postgres
 
 | Name | Description | Type | Default | Required | Notes | Actions |
 | :--- | ---: | ---: | ---: | ---: | ---: | ---: |
-| V4_CFG_POSTGRES_ADMIN_LOGIN | Existing postgres username | string | | true | | vdm |
-| V4_CFG_POSTGRES_PASSWORD | Existing postgres password | string | | true | | vdm |
-| V4_CFG_POSTGRES_FQDN | Existing postgres ip/fqdn | string | | true | | vdm |
-| V4_CFG_POSTGRES_PORT | Existing postgres port | string | 5432 | false | | vdm |
-| V4_CFG_POSTGRES_DATABASE | Existing postgres database name | string | "SharedServices" | false | | vdm |
+| V4_CFG_POSTGRES_ADMIN_LOGIN | Existing postgres username | string | | true | | viya |
+| V4_CFG_POSTGRES_PASSWORD | Existing postgres password | string | | true | | viya |
+| V4_CFG_POSTGRES_FQDN | Existing postgres ip/fqdn | string | | true | | viya |
+| V4_CFG_POSTGRES_PORT | Existing postgres port | string | 5432 | false | | viya |
+| V4_CFG_POSTGRES_DATABASE | Existing postgres database name | string | "SharedServices" | false | | viya |
 
 ## LDAP / Consul
 
 | Name | Description | Type | Default | Required | Notes | Actions |
 | :--- | ---: | ---: | ---: | ---: | ---: | ---: |
-| V4_CFG_EMBEDDED_LDAP_ENABLE | Deploy openldap in the namespace for authentication | bool | false | false | [Openldap Config](../roles/vdm/templates/generators/openldap-bootstrap-config.yaml) | vdm |
-| V4_CFG_CONSUL_ENABLE_LOADBALANCER | Setup LB to access consul ui | bool | false | false | Consul ui port is 8500 | vdm |
+| V4_CFG_EMBEDDED_LDAP_ENABLE | Deploy openldap in the namespace for authentication | bool | false | false | [Openldap Config](../roles/viya/templates/generators/openldap-bootstrap-config.yaml) | viya |
```

Comment:
You also changed the directory here was the intentional?

Reply:
No, that was not intentional

Reply:
I'll wait for the update, then re-check.

Reply:
updated